www/authoring/item-list-enable-selection.php

```diff
@@ -72,7 +72,7 @@
         <div class="overview">
             <h2>Select Multiple Items from Item Browsing</h2>
             <p>The item list mode allows users to choose and select multiple items. This is particularly useful when building an item picker as part of a larger solution.<br>
-                Your application can then get the array of selected items using the <a href="https://reference.learnosity.com/author-api/methods#getSelectedItems">getSelectedItems()</a> method.</p>
+                Your application can then get the array of selected items using the <a href="https://reference.learnosity.com/methods/authorApp/getSelectedItems">getSelectedItems()</a> method.</p>
```

Comment:
This link isn't working. Might need to be:
https://reference.learnosity.com/author-api/methods/authorApp/getSelectedItems

Reply:
Thanks Kent, I corrected it in here: https://github.com/Learnosity/learnosity-demos/pull/229/commits/7adc7c566443292e407687d8ca1dba3cf888ac6f